docs/csharp/language-reference/compiler-options/how-to-set-environment-variables-for-the-visual-studio-command-line.md

```diff
@@ -40,20 +40,24 @@ translation.priority.ht:
   - "zh-tw"
 ---
 # How to: Set Environment Variables for the Visual Studio Command Line
-The vsvars32.bat file sets the appropriate environment variables to enable command-line builds. For more information about vsvars32.bat, see [Knowledge Base article Q248802](http://go.microsoft.com/fwlink/?LinkId=225042).  
+
+The VsDevCmd.bat file sets the appropriate environment variables to enable command-line builds. For more information about VsDevCmd.bat, see [Knowledge Base article Q248802](http://go.microsoft.com/fwlink/?LinkId=225042).  
+
+> [!NOTE]
+> The VsDevCmd.bat file is a new file delivered with Visual Studio 2017. Visual Studio 2015 and earlier versions used VSVARS32.bat for the same purpose. This file was stored in \Program Files\Microsoft Visual Studio\\*Version*\Common7\Tools or Program Files (x86)\Microsoft Visual Studio\**Version*\Common7\Tools.
```

Comment:
In the second version, you've added an asterisk instead of a backlash. (`\**Version` --> `\\*Version`)

Reply:
Just pushed a fix. I'll check it on the review site once it builds.